index.php

```diff
@@ -207,71 +161,62 @@ class="btn-large waves-effect waves-light light blue lighten-1" style="border-ra
 	</div>
 </div>
 	
-	<div class="flow-text" style="background-color: #FFF5EE; display: flex">
+	<div class="flow-text section" style="background-color: #FFF5EE; display: flex">
 	<div class="container">
-	<div class="center" style="padding:1% 0 1% 0"><h3>DEMOCRACY DISRUPTS EXTREME POVERTY.</h3> <br>Nearly 400 million people (and growing) live in extreme poverty in rural Africa. 
-	We're changing that with a model that celebrates village democracy, direct giving, and data analysis. Small transfers of cash make a big difference
-	when they target underserved villages on the ground. Read about us in <a href="https://www.nytimes.com/2018/07/17/opinion/development-aid-liberia.html" target="_blank">The New York Times</a>.</div><br>
+	<div class="center" style="padding:1% 0 1% 0"><h3>DEMOCRACY DISRUPTS EXTREME POVERTY</h3> <br>Nearly 400 million people (and growing) live in extreme poverty in rural Africa. 
+	We're changing that with a model that celebrates <b>village democracy, direct giving, and data analysis</b>. Small transfers of cash make a big difference
+	when they target public goods chosen by underserved villages. Read about us in <a href="https://www.nytimes.com/2018/07/17/opinion/development-aid-liberia.html" target="_blank">The New York Times</a>.</div><br>
 	</div>
 	</div>
 
-<div class="container hide-on-med-and-down"">
-			<h4 class="header center light blue-text text-lighten-2" style="padding:1% 0 0 0">How It Works</h4>
+<div class="container">
+			<h4 class="header center light blue-text text-lighten-2" style="text-transform:uppercase; padding:1% 0 0 0">How It Works</h4>
 
 				<div class="row">
-					<div class="col s12 m4">
+					<div class="col s12 m12 l4">
 						<div class="icon-block">
 							<h2 class="center brown-text">
 							  <source type="image/webp" srcset="images/how_it_works_democracy.webp">
 							  <source type="image/jpeg" srcset="images/how_it_works_democracy.jpg">
 							  <img style="border:5px solid rgba(0, 0, 0, .85);" class="circle responsive-img" src="images/how_it_works_democracy.jpg">
 							</h2> 
 							
-							<h5 class="center flow-text" style="font-weight: 600;">Villages choose projects.</h5>
-
-							<p class="light center">Search the projects page or interactive map for villages battling extreme poverty in rural
-								Africa. Find a village-led development project that changes lives and speaks to you.</p>
-								
-							<h5 class="center">#localdemocracy</h5>
+							<h5 class="center flow-text" style="font-weight: 300;padding:0% 0 2% 0%">Bakili Village votes to build a nursery school to improve early childhood education.</h5>
+			
+							<h5 class="center">#democracy</h5> 
 
 						</div>
 					</div>
 
-					<div class="col s12 m4">
+					<div class="col s12 m12 l4">
 						<div class="icon-block">
 							<h2 class="center brown-text">
 								<picture>
-								  <source type="image/webp" srcset="images/how_it_works_directgiving.webp">
```

Comment:
You can just replace the whole picture block with the one img tag.  Maybe, instead of calling it directgiving_2, you can just delete the old directgiving and name it that?